third-party/devtools-tests/e2e/lighthouse/navigation_test.ts

```diff
@@ -35,9 +62,10 @@ describe('Navigation', async function() {
       });
 
       it('successfully returns a Lighthouse report', async () => {
-        await navigateToLighthouseTab('lighthouse/hello.html');
+        await setDevToolsSettings({language: 'en-XL'});
```

Comment:
Why modify the existing tests? Couldn't we add another i18n specific test?

Reply:
Because these tests are slow and expensive to run.

Reply:
I think this is better suited for the "test lh with flags changed" test that was added recently. I think we need to do a test sync for that though.

Reply:
(spoke offline) we gonna move this around later